components/desktop/e/enlightenment/enlightenment.p5m

```diff
@@ -22,7 +22,6 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-<transform file path=usr/lib/(.*)/module.so$ -> default mode 0555>
 <transform file path=usr/lib/$(MACH64)/enlightenment/utils/(.*)$ -> default mode 0555>
 
 file path=usr/bin/emixer
```

Comment:
At least, while you are here, fix usr/etc directory location, likely application needs sysconfdir